monai/bundle/scripts.py

```diff
@@ -717,9 +719,18 @@ def run(
     )
     if "config_file" not in _args:
         warnings.warn("`config_file` not provided for 'monai.bundle run'.")
+    if meta_file is None:
+        _args["meta_file"] = None  # None input is ignored in "_update_args"
+    if logging_file is None:
+        _args["logging_file"] = None
     _log_input_summary(tag="run", args=_args)
     config_file_, meta_file_, runner_id_, logging_file_, tracking_ = _pop_args(
-        _args, config_file=None, meta_file=None, runner_id="", logging_file=None, tracking=None
+        _args,
+        config_file=None,
+        meta_file="configs/metadata.json",
+        runner_id="",
+        logging_file="configs/logging.conf",
```

Comment:
I think line 737 FileNotFoundError is too restrictive, the training can continue without a logging file.

Reply:
Hi @wyli ,

The bundle can run without logging file, I think this line only raise error when a user actually set the logging config file path and the file doesn't exist.

Thanks.

Reply:
now the default is `configs/logging.conf`, I also hope it's ok if by default I don't have this file in a folder...how about raising a warning message if it's the default value and the default config doesn't exist?

Reply:
A warning sounds good to me.

Thanks.

Reply:
Thanks @wyli , it makes sense. I just modified according to the comments.